dist/index.html

```diff
@@ -10,13 +10,26 @@
   <section class="all-content">
     <section class="header-section">
       <header>
-        <h1 class= "welcome-message" id="welcomeMessage">Welcome!</h1>
-        <img class="logo" src="Wistledown-Logo.png" alt="Wistledown inn logo with a bee surrounded by flowers">
+        <h1 class= "welcome-message hidden" id="welcomeMessage">Welcome!</h1>
+        <img class="logo" src="Whistledown-Logo.png" alt="Wistledown inn logo with a bee surrounded by flowers">
       </header>
-      <div class="header-image-container">
+      <div class="header-image-container user-login-container">
```

Comment:
Right now, the file path for the `background image` is set in the `css file` using a `relative path` (`/Users/adelle_pitsas/turing_work/2mod/projects/overlook-hotel/dist/BrigertonHouse.png`), however that path will only work locally since it's made for your machine. 

If you comment out that line in your css and add `style="background-image: url('BrigertonHouse.png')` to `line 16` that will give the background image for everyone. 

Not sure inline css styling in html is the best solution, but I think you would need to redo the way you do your images to get it to work through the `css file`. Instead of putting all your images in dist, have them in `images` folder, import them into your `scripts.js`, and set your `src` to the images folder like the `turing logo example`.  Not sure it would be worth doing all that though for just the background image.

Reply:
thanks for this! I'm still trying to figure out how file paths work! That makes sense about the local path. I've moved the images into the images folder and imported them into my `scripts` file!

Reply:
Just checked out and your image imports look great!